Running `just frb` fails locally and in CI

```
Compiling plist v1.7.0
error[E0283]: type annotations needed
   --> /Users/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/plist-1.7.0/src/stream/binary_reader.rs:252:58
    |
252 |                 if value < 0 || value > u64::max_value().into() {
    |                                       -                  ^^^^
    |                                       |
    |                                       type must be known at this point
    |
    = note: multiple `impl`s satisfying `i128: PartialOrd<_>` found in the following crates: `core`, `deranged`:
            - impl PartialOrd for i128;
            - impl<MIN, MAX> PartialOrd<deranged::RangedI128<MIN, MAX>> for i128
              where the constant `MIN` has type `i128`, the constant `MAX` has type `i128`;
help: try using a fully qualified path to specify the expected types
    |
252 |                 if value < 0 || value > <u64 as Into<T>>::into(u64::max_value()) {
    |                                         +++++++++++++++++++++++                ~

error[E0283]: type annotations needed
   --> /Users/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/plist-1.7.0/src/stream/binary_reader.rs:[252](https://github.com/breez/breez-sdk-liquid/actions/runs/14027644513/job/39269040665?pr=782#step:9:253):58
    |
252 |                 if value < 0 || value > u64::max_value().into() {
    |                                                          ^^^^
    |
note: multiple `impl`s satisfying `_: From<u64>` found
   --> /Users/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/plist-1.7.0/src/integer.rs:91:1
    |
91  | impl From<u64> for Integer {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
   ::: /Users/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/plist-1.7.0/src/value.rs:552:1
    |
552 | impl From<u64> for Value {
    | ^^^^^^^^^^^^^^^^^^^^^^^^
    = note: and more `impl`s found in the following crates: `core`:
            - impl From<u64> for AtomicU64;
            - impl From<u64> for i128;
            - impl From<u64> for u128;
    = note: required for `u64` to implement `Into<_>`
help: try using a fully qualified path to specify the expected types
    |
252 |                 if value < 0 || value > <u64 as Into<T>>::into(u64::max_value()) {
    |                                         +++++++++++++++++++++++                ~

For more information about this error, try `rustc --explain E0283`.
error: could not compile `plist` (lib) due to 2 previous errors
```

--- a/lib/bindings/langs/flutter/justfile
+++ b/lib/bindings/langs/flutter/justfile
@@ -21,7 +21,8 @@ clean:
 
 # Install flutter_rust_bridge_codegen dependencies
 frb:
-	cargo install cargo-expand
+	# This is locked plist PR https://github.com/ebarnard/rust-plist/pull/152 is merged/released
+	cargo install cargo-expand --locked
 	cargo install flutter_rust_bridge_codegen --version 2.9.0
 	dart pub global activate ffigen
 	dart pub global activate ffi

